releasenotes/source/index.rst

```diff
@@ -0,0 +1,14 @@
+==========================================================
+Rackspace Private Cloud Powered By OpenStack Release Notes
+==========================================================
```

Comment:
This reads to me like 'Rackspace Private Cloud'...'Powered By'...'Openstack Release Notes'. Maybe a dash before 'Release Notes' ?